frontends/web/src/routes/settings/settings.tsx

```diff
@@ -354,6 +354,21 @@ class Settings extends Component<Props, State> {
                                                         </div>
                                                     </div>
                                                     <div className="box slim divide">
+                                                        <div className={style.currency}>
+                                                            <div>
+                                                                <p className="m-none">{t('settings.expert.splitAccounts')}</p>
+                                                                <p className="m-none">
+                                                                    <Badge type="primary">BB02</Badge>
+                                                                    <Badge type="secondary" className="m-left-quarter">BB02-BTC</Badge>
+                                                                    <Badge type="generic" className="m-left-quarter">BTC</Badge>
+                                                                    <Badge type="generic" className="m-left-quarter">LTC</Badge>
+                                                                </p>
+                                                            </div>
+                                                            <Toggle
+                                                                id="splitAccounts"
+                                                                checked={config.backend.splitAccounts}
+                                                                onChange={this.handleToggleAccount} />
```

Comment:
How do I find this handler, where is it?

Reply:
https://github.com/digitalbitbox/bitbox-wallet-app/blob/5813f002a37a06cf8ef710add60d263e0f1787cd/frontends/web/src/routes/settings/settings.tsx#L102

Reply:
oh, it's reusing an existing function. sorry for my ignorance.

Reply:
Further up in the same class:

https://github.com/digitalbitbox/bitbox-wallet-app/blob/master/frontends/web/src/routes/settings/settings.tsx#L102

Calling `POST/GET /config` in backend/handlers.go.

Reply:
yeah, somehow I thought it had to be a new handler func. not sure why.